importer/Cargo.toml

```diff
@@ -21,3 +21,4 @@ rand_xorshift = "0.2.0"
 serde = "1.0.110"
 serde_json = "1.0.40"
 sim = { path = "../sim" }
+geojson = "0.19"
```

Comment:
Ah, please keep sorted. I wish rustfmt would do this.

Reply:
whoops! Fixed.